README.md

```diff
@@ -1,2 +1,17 @@
 # FusionIIIT  
-Coming Soon
+Software Engineering project to be deployed int the institute for the automation of various functionalities, modules and tasks of the Institute that are currently being handled either manually or ineffeciently.
+A team of more than 60 people working in 12 groups with each working on a module assigned to them.
+The automated tasks includes various sections of the institute like:
+    #1) Academic database management
```

Comment:
In markdown files you need to provide two spaces at the end of line for the next line to show up in next line.  
Life do this:  
```
+ Line 1<space><space><enter>  
+ Line 2<space><space><enter>  

Output:
+ Line 1
+ Line 2

Reply:
ok, will change  it